lib/simplecov.rb

```diff
@@ -53,6 +53,17 @@ def start(profile = nil, &block)
       Coverage.start
     end
 
+    #
+    # Method added by NREL to shutdown SimpleCov and collect results
+    # this will set running to false which will prevent from running again at_exit
+    def end_now
+      SimpleCov.set_exit_exception
+      exit_status = SimpleCov.exit_status_from_exception
+      SimpleCov.result.format!
+      exit_status = SimpleCov.process_result(SimpleCov.result, exit_status)
```

Comment:
I think `SimpleCov` should be `self` here so no need to address it directly imo. Also I think the last assignment and return aren't needed :)